lib/src/request.dart

```diff
@@ -13,14 +13,12 @@ import 'message.dart';
 import 'util.dart';
 
 /// A callback provided by a Shelf handler that's passed to [Request.hijack].
-@Deprecated("Will be removed in shelf 0.7.0.")
-typedef void HijackCallback(
+typedef void _HijackCallback(
```

Comment:
Wouldn't you want to make this take a StreamChannel, too?

Reply:
I can if I change here: https://github.com/dart-lang/shelf/blob/17182d1da9a25768a66a45b87e94a65f6809f91a/lib/src/request.dart#L146

Want me to do that?

Reply:
Seems reasonable, right? Keep 'em consistent? If we're forcing one end to use StreamChannel, the user has already dealt with the extra class/library.

Am I missing anything?

Reply:
Done